app/assets/src/components/common/project_creation_form.scss

```diff
@@ -5,6 +5,13 @@
   .label {
     @include font-header-s;
     margin-bottom: 5px;
+
+    .infoLink {
+      @include font-label-s;
+      margin-left: 10px;
```

Comment:
We have some spacing variables in `_elements.scss` that could be used here 😄 (`$space-default` for 10px)